lib/mix/lib/mix/release.ex

```diff
@@ -802,4 +809,16 @@ defmodule Mix.Release do
         error
     end
   end
+
+  @spec parse_strip_beams_option(true | false | Keyword.t()) :: {true | false, List.t() | nil}
+  def parse_strip_beams_option(option) do
+    {strip?, exceptions} =
+      case option do
+        [keep: exceptions] when is_list(exceptions) -> {true, exceptions}
+        true -> {true, []}
+        false -> {false, nil}
+      end
+
+    {strip?, @significant_chunks ++ exceptions}
```

Comment:
I think we are adding the `@significant_chunks` twice, here and in `strip_beam/1,2`